task/testmap.py

```diff
@@ -136,16 +136,12 @@
     },
     'weldr/cockpit-composer': {
         'master': [
-            'fedora-30/chrome',
-            'rhel-8-2/firefox',
+            'fedora-31/chrome',
             'fedora-31/edge',
-            'fedora-31/firefox',
-            'rhel-7-8/firefox',
-            'rhel-8-1/chrome',
+            'fedora-31/firefox',  
```

Comment:
There are trailing spaces an linter is complaining, can you please fix?
(added no-test label, so we can run only what we need on the next push force)

Reply:
Thanks for pointing this out. Should be fixed now